### DISCUSSION TITLE
How can I use a custom color for the ProgressFilledTrack in the Progress component?

### DISCUSSION THREAD

**Discussion by Tisks** [chakra-ui/chakra-ui#7043] [Q&A]
Does anyone know?
The default option is the `ColorScheme` but I want to use a custom color in hex or even one that is in my theme but is not the default one.

I have this in my colors const
```
teal: {
    50: '#E9F1FF',
    100: '#d1f1f0',
    200: '#67d1ce',
    300: '#02b0ad',
    400: '#007a87',
    500: '#007187',
  },
```

But if I use `colorScheme='teal'` it takes the `teal.500`.
I want to use `teal.300`.

**Comment by spidertwin2** [comment]
I have the same problem.  Unable to change fill colour of `Progress`.

`<Progress colorScheme="orange"/>`  uses `orange.500`, but I want to change it to `orange.200`.

Can someone help with this please 🙏

**Reply by nataliadavtyan** [reply]
I found this reply to be helpful:

https://stackoverflow.com/questions/71011761/how-to-change-the-background-color-of-the-chakra-ui-progress-component
